crates/spicepod/src/component.rs

```diff
@@ -13,7 +13,7 @@ pub trait WithDependsOn<T> {
 
```

Comment:
Looks like this needs some more work, the CLI is now broken.

Running:

`spice init`
`spice dataset configure`
`spiced`

Results in:

```
src/mitchdevenport/test-app ❯ spiced
2024-01-31T16:38:46.831561Z TRACE spiced: Starting Spice Runtime!
2024-01-31T16:38:46.834285Z ERROR spiced: Spice Runtime error: UnableToConstructSpiceApp { source: UnableToLoadSpicepod { source: UnableToParseSpicepod { source: Error("datasets: data did not match any variant of untagged enum ComponentOrReference", line: 5, column: 1) }, path: "." } }
```

Reply:
I'm working on a PR to fix this